### PR TITLE
AA-255: Dates Tab Tooltip for ORA

### PR DESCRIPTION
**Pull request by ciduarte**
Screen Shot 2020-08-10 at 9 15 37 AM

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -4911,9 +4911,9 @@
       "dev": true
     },
     "bootstrap": {
-      "version": "4.5.0",
-      "resolved": "https://registry.npmjs.org/bootstrap/-/bootstrap-4.5.0.tgz",
-      "integrity": "sha512-Z93QoXvodoVslA+PWNdk23Hze4RBYIkpb5h8I2HY2Tu2h7A0LpAgLcyrhrSUyo2/Oxm2l1fRZPs1e5hnxnliXA=="
+      "version": "4.5.1",
```
Comment (mikix) [inline package-lock.json]: Did you need 4.5.1 or was this an accidental change?

File: src/course-home/dates-tab/Day.jsx [hunk 2]
```diff
@@ -3,6 +3,9 @@ import PropTypes from 'prop-types';
 import classNames from 'classnames';
 import { useSelector } from 'react-redux';
 import { FormattedDate, injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+import { Tooltip, OverlayTrigger } from '@edx/paragon';
```
Comment (mikix) [inline src/course-home/dates-tab/Day.jsx:6]: Oh nice, did those get added to Paragon as part of the react-bootstrap pull-ins?

**Review by mikix — APPROVED** [review]

File: src/course-home/dates-tab/Day.jsx (hunk 2 above)
Comment (ciduarte) [reply src/course-home/dates-tab/Day.jsx:6]: Yes these got added in an earlier release!

File: package-lock.json (hunk 1 above)
Comment (ciduarte) [reply package-lock.json]: Ooh this was unintentional

**Pull request merged**